docs/generated/eventlog.md

```diff
@@ -2201,6 +2201,64 @@ An event of type `drop_role` is recorded when a role is dropped.
+telemetry channel. It contains usage details about the parameters the user
+passed in to define the changefeed
+
+
+| Field | Description | Sensitive |
+|--|--|--|
+| `NumTables` | The number of tables listed in the query that the changefeed is to run on | no |
+| `TopicPrefix` | (Kafka/CloudStorage Query Param) Whether a custom prefix is being added to all topic names | no |
+| `TlsEnabled` | (Kafka Query Param) Whether TLS is enabled on the connection to Kafka | no |
+| `CaCert` | (Kafka Query Param) The base64-encoded ca_cert file | no |
+| `ClientCert` | (Kafka Query Param) Whether a Privacy Enhanced Email (PEM) certificate was specified | no |
+| `ClientKey` | (Kafka Query Param) Whether a private key for the PEM certificate was specified | no |
+| `SaslEnabled` | (Kafka Query Param) Whether SASL is enabled | no |
+| `SaslMechanism` | (Kafka Query Param) The SASL mechanism (ex: SASL-SCRAM-SHA-256, SASL-PLAIN) | yes |
+| `SaslUser` | (Kafka Query Param) Whether a SASL username has been specified | no |
+| `SaslPassword` | (Kafka Query Param) Whether a SASL password has been specified | no |
+| `FileSize` | (CloudStorage Query Param) A custom maximum file size for files before they are flushed | yes |
+| `InsecureTlsSkipVerify` | (Kafka Query Param) Whether client-side validation of responses has been disabled | no |
+| `Updated` | (Changefeed Option) Whether updated timestamps are emitted with each row | no |
+| `Resolved` | (Changefeed Option) The interval at which resolved timestamps are emitted | yes |
+| `Envelope` | (Changefeed Option) Either key_only to emit only keys or wrapped for both key and value | yes |
+| `Cursor` | (Changefeed Option) Whether a cursor timestamp has been specified to begin emitting events from | no |
+| `Format` | (Changefeed Option) The data format, either JSON or Avro, that the changefeed emits | yes |
+| `ConfluentSchemaRegistry` | (Changefeed Option) Whether a schema registry address has been specified | no |
+| `KeyInValue` | (Changefeed Option) Whether to emit primary keys as the value for deleted rows | no |
+| `Diff` | (Changefeed Option) Whether a `before` field is to be emitted with each message | no |
+| `Compression` | (Changefeed Option) The compression format being used (ex: gzip) | yes |
+| `ProtectDataFromGcOnPause` | (Changefeed Option) Whether data to resume a changefeed is protected from GC when paused | no |
+| `SchemaChangeEvents` | (Changefeed Option) The type of schema events that trigger the change policy | yes |
+| `SchemaChangePolicy` | (Changefeed Option) The behavior to trigger from a schema change event (ex: backfill) | yes |
+| `Scan` | Whether the initial_scan or no_initial_scan options have been specified | yes |
+| `FullTableName` | (Changefeed Option) Whether fully-qualified table names are used for topic names | no |
+| `AvroSchemaPrefix` | (Changefeed Option) Whether a custom namespace for table schemas is specified | no |
```

Comment:
how did we determine which fields are sensitive and which aren't? `Scan` marked sensitive, while `CaCert` marked as not sensitive -- and I'm confused about those choices.

I'm also not clear at all about which fields do we think it makes sense to log from the product perspective.
In particular, do we really care about the 4 Sasl fields specified, or do we just care that SASL was used?
Do we care about fileSize values, or do we care if file size was specified?
@amruss -- can you chime in?

Reply:
I think it just labels string parameters as sensitive and bool parameters as not sensitive.  I manually added a tag to mark them not-sensitive.

Reply:
I guess I want to know what exactly is the guidance here?  We do have bunch of security related fields that are clearly sensitive, yet are not marked as such.

Reply:
The idea here was anything sensitive we'd just have as a `bool` saying whether it was specified or not, while for some non-sensitive parameters we'd send over the full string, so currently the only string values we send are:
- sasl-mechanism
- file_size
- resolved
- envelope
- compression
- schema_change_events
- schema_change_policy
- scan

Those are all basically just enums / a number rather than user info